src/__mocks__/api-handlers.js

```diff
@@ -137,4 +135,31 @@ export const handlers = [
         ctx.status(httpOk),
     )
   }),
+
```

Comment:
Mock responses should resemble the real API shape and response as much as possible.